plugins/formal-verification/testData/diagnostics/no_contracts/class_constructors.kt

```diff
@@ -0,0 +1,46 @@
+class Foo {
+    val a: Int
+    constructor(n: Int) {
+        this.a = n
+    }
+    constructor(b: Boolean) {
+        this.a = when (b) {
+            true -> 1
+            false -> 0
+        }
+    }
+
+    constructor(x1: Int, x2: Int) {
+        a = x1 + x2
+    }
+}
+
+class Bar(val a: Int) {
+    constructor(b: Boolean) : this(when (b) {
+                                        true -> 1
+                                        false -> 0
+                                    }) {
```

Comment:
Nit: The formatting here seems a little off. I that what the kotlin linter recommends?

Reply:
My IntelliJ doesn't run the linter on the Kotlin test files 🤔
I can try to simplify it with an `if` statement!